DependencyInjection/CompilerPass/LoggerAwarePass.php

```diff
@@ -1,54 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace ETSGlobal\LogBundle\DependencyInjection\CompilerPass;
-
-use Psr\Log\LoggerAwareInterface;
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Reference;
-
-/**
- * Automatically inject logger service into tagged services.
- *
- * Finds all tagged services and add logger by setter injection
- * on classes that implements Psr\Log\LoggerAwareInterface.
- *
- * @internal
- */
-final class LoggerAwarePass implements CompilerPassInterface
```

Comment:
:+1: not needed since Symfony 4.2 https://github.com/symfony/symfony/commit/afda3c8844a78b47f4229793cc351f637d3dfae1